src/parser.js

```diff
@@ -734,8 +736,29 @@ export default class Parser {
         transition.id = methodValue.operationId;
       }
 
+      /* eslint-disable prefer-const */
+      let resourceParametersWithPath = [];
+      /* eslint-enable prefer-const */
+
+      _.forEach(resourceParameters, (resourceParameter, index) => {
+        let inTransition = false;
+        const len = transitionParameters.length;
+
+        for (let i = 0; i < len; i++) {
```

Comment:
Actually, I think it would be simpler to use a [`some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) for the matching parameter. There would be less state to track. 

Something like:

```js
const hasTransitionParameter = transitionParamters.some((parameter) => parameter.name == resourceParameter.name...);

if (!hasTransitionParameter) {
    // push param
} 
```